modules/ROOT/pages/_partials/cookbook-dw/rename-keys-ex02/transform.dwl

```diff
@@ -0,0 +1,13 @@
+%dw 2.0
+output application/json
+---
+payload.flights map (flight) ->
+flight mapObject (value, key) -> {
+ ((key) match
+		{
+			case literalMatch: "availableSeats" -> "emptySeats"
+			case literalMatch: "airlineName" -> "airline"
```

Comment:
I wouldn't name the cases since that variable isn't used later

I'd leave it as
```
case "availableSeats" -> "emptySeats"
case "airlineName" -> "airline"
```